src/BaseServiceProvider.php

```diff
@@ -28,6 +29,8 @@ public function boot(\Illuminate\Routing\Router $router)
         // - then the stock views that come with the package, in case a published view might be missing
         $this->loadViewsFrom(realpath(__DIR__.'/resources/views'), 'backpack');
 
+        View::composer('backpack::*', \Backpack\Base\app\Http\ViewComposers\AuthComposer::class);
```

Comment:
@jorenvanhee - I'm wondering if this is broad enough. Wouldn't this make the ```$backpackAuth``` guard available only in views that are loaded within the backpack namespace?

If so, I see the following problem:
- developer uses $backpackAuth when he overwrites backpack views;
- developer create supplementary views, that he considers to be part of the admin panel; he tries to use $backpackAuth there too; he expects ```$backpackAuth``` to be there too, but it's not;

I may be mistaken, though. It's been known to happen :-)